Applications/CommandLineInterface/src/selxConfigurationReader.cxx

```diff
@@ -21,12 +21,13 @@
 #define ConfigurationReader_cxx
 
 #include "selxConfigurationReader.h"
+#include "selxLoggerImpl.h"
 #include <ostream>
 
 namespace selx
 {
 ConfigurationReader::ParameterValueType
-ConfigurationReader::VectorizeValues( const ComponentOrConnectionTreeType & componentOrConnectionTree )
+ConfigurationReader::VectorizeValues( ComponentOrConnectionTreeType & componentOrConnectionTree )
```

Comment:
Why is this const dropped?

Reply:
Clang warned it did not have any effect and so it is misleading

Reply:
fair